drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -1547,11 +1548,16 @@ function DragAndDropBlock(runtime, element, configuration) {
                     var dx = x - drag_origin.x;
                     var dy = y - drag_origin.y;
 
+                    if (typeof item.max_left === 'undefined') {
+                        var $item = $container.find('.option[data-value=' + item_id + ']');
```

Comment:
$item is already a variable and this will shadow that, perhaps call this $dragged_item.